tests/functional/test_accounts.py

```diff
@@ -676,3 +683,46 @@ def test_load_public_key_from_keyfile(runner, keyfile_account):
         )
```

Comment:
Stuff like `="a\n` in above input to tests can be replaced, maybe we should make a passphrase fixture or constant

Reply:
Yeah, working on making these a bit more resilient.  Just been proving tedious running each failing test down.